Makefile

```
@@ -0,0 +1,4 @@
+all: .gitmodules
+
+.gitmodules:
+	./git-setup.sh api website console
```

It would add(try to add) all three submodules for non-core team members as well. Shouldn't we let the core team members run this command manually?

---

It will fail silently without disrupting anything. Did you try this locally?